src/webview_provider/queryResultPanel.ts

```diff
@@ -85,10 +98,12 @@ export class QueryResultPanel implements WebviewViewProvider {
 
   private _disposables: Disposable[] = [];
   private _panel: WebviewView | undefined;
+  private adapter: string = "";
```

Comment:
empty string is not really clean. Should be rather undefined.

Reply:
setting it to unknown to match dbtProject.ts